test/testdata/README.md

```diff
@@ -8,196 +8,3 @@ $ openssl req -key test/testdata/test_blob_private_key -x509 -days 3650 -out cer
 ```
 
 and then you replace the old `test/testdata/test_blob_cert.pem` with the new certificate.
```

Comment:
Looking at [this test](https://github.com/sigstore/cosign/blob/302aee6c1648e5a85f77f785ca351f62d573a5e7/test/sign_blob_test.sh#L17-L20) again, this sounds like a test that should be done in e2e testing rather than its own script. So we can reuse what's in e2e_tests already.